lib/main/bindings/network/network_isolate_binding.dart

```diff
@@ -0,0 +1,70 @@
+import 'package:core/core.dart';
+import 'package:dio/dio.dart';
+import 'package:flutter/foundation.dart';
+import 'package:get/get.dart';
+import 'package:jmap_dart_client/http/http_client.dart' as JmapHttpClient;
+import 'package:tmail_ui_user/features/email/data/network/email_api.dart';
+import 'package:tmail_ui_user/features/login/data/local/account_cache_manager.dart';
+import 'package:tmail_ui_user/features/login/data/local/token_oidc_cache_manager.dart';
+import 'package:tmail_ui_user/features/login/data/network/authentication_client/authentication_client_base.dart';
+import 'package:tmail_ui_user/features/login/data/network/config/authorization_interceptors.dart';
+import 'package:tmail_ui_user/features/mailbox/data/network/mailbox_isolate_worker.dart';
+import 'package:tmail_ui_user/features/thread/data/network/thread_api.dart';
+import 'package:tmail_ui_user/features/thread/data/network/thread_isolate_worker.dart';
+import 'package:tmail_ui_user/main/bindings/network/binding_tag.dart';
+import 'package:worker_manager/worker_manager.dart';
+
+class NetworkIsolateBindings extends Bindings {
+
+  @override
+  void dependencies() {
+    _bindingDio();
+    _bindingApi();
+    _bindingIsolateWorker();
+  }
+
+  void _bindingDio() {
+    final dio = Get.put(Dio(Get.find<BaseOptions>()), tag: BindingTag.isoLateTag);
+    Get.put(DioClient(dio), tag: BindingTag.isoLateTag);
+    _bindingInterceptors(dio);
+  }
+
+  void _bindingInterceptors(Dio dio) {
+    Get.put(AuthorizationInterceptors(
+      dio,
+      Get.find<AuthenticationClientBase>(),
+      Get.find<TokenOidcCacheManager>(),
+      Get.find<AccountCacheManager>(),
+    ), tag: BindingTag.isoLateTag);
+    dio.interceptors.add(Get.find<DynamicUrlInterceptors>());
```

Comment:
Why not put `DynamicUrlInterceptors ` in `isolateTag`

Reply:
DynamicUrlInterceptors already include network_binding and there is no need to create a new DynamicUrlInterceptors to use for dio of isolate. So I not put DynamicUrlInterceptors in isolateTag

Reply:
> no need to create a new DynamicUrlInterceptors to use for dio of isolate

I think it is different instance? Why?